content/en/agent/guide/_index.md

```diff
@@ -20,6 +20,7 @@ disable_toc: true
     {{< nextlink href="agent/guide/python-3" >}}Python 2 to 3 Custom Check Migration{{< /nextlink >}}
     {{< nextlink href="agent/guide/cluster-agent-custom-metrics-server" >}}Cluster Agent and Horizontal Pod Autoscaling: Custom Metrics Server{{< /nextlink >}}
     {{< nextlink href="agent/guide/autodiscovery-with-jmx" >}}Autodiscovery with JMX{{< /nextlink >}}
+    {{< nextlink href="agent/guide/private-link" >}}Forward your logs from the AWS us-east-1 region to Datadog with a PrivateLink{{< /nextlink >}}
```

Comment:
reminder: AWS branding is AWS PrivateLink.  this feature is more more data types than just logs.